recipe/meta.yaml

```diff
@@ -12,29 +12,16 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
   skip: True  # [py2k and py<34]
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - python
     - setuptools
-    - pandas
-    - numpy >=1.12
-    - scikit-learn
-    - matplotlib
-    - jupyter
-    - ipywidgets
-    - ipython
   run:
     - python
-    - numpy >=1.12
-    - scikit-learn
-    - matplotlib
-    - jupyter
-    - ipywidgets
-    - ipython
```

Comment:
@ocefpaf these packages are needed ... 

But in setup.py I don't import this packages*, should I keep these packages in requirements?

* https://github.com/conda-forge/staged-recipes/pull/3247#discussion_r126273270)